test/CCR/NTTtransfer/NttTransferTest.t.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import { BaseCCRTest } from "../BaseCCR.t.sol";
+
+import { console } from "forge-std/console.sol";
+
+import {WormholeSimulator} from "wormhole-solidity-sdk/testing/helpers/WormholeSimulator.sol";
+import {TransceiverStructs} from "./../../../contracts/libraries/wormhole-lib/TransceiverStructs.sol";
+import "./../../../contracts/libraries/wormhole-lib/TrimmedAmount.sol";
+contract CreateChannelCCR is BaseCCRTest {
+    // using TrimmedAmount for TrimmedAmount;
+    uint256 constant DEVNET_GUARDIAN_PK =
+        0xedc2d60cdb193aac203bea0be0f5f1b016bf4381f92231ca0320fc01a57bcae5;
+    WormholeSimulator guardian;
+
+    function setUp() public override {
+        BaseCCRTest.setUp();
+        sourceAddress = toWormholeFormat(address(commProxy));
+
+    }
+    function test_tokenTransferSuccessful() external {
```

Comment:
feels a bit weird about ntt transfer tests
if you see this ccr tx: https://sepolia.basescan.org/tx/0x244d7c67dd4e8512e7d606f2455863d456460206359dfd52f080524bf720f143#eventlog

it has a separate LogMessagePublished and TransferSent event specific for ntt transfer 
try to look at it, and if it can be used to fetch the values and then call receiveWormholeMessages fn on destination chain using those variable
It'd be more robust if this could be achieved

Reply:
Done @0xNilesh 

Using `TransferSent` event as of now.